Bottom: f2e778448bdc99d59e0c4ccc6324a6af6fd14d9d
Top:    cb812a54749f94032c0f12e107d7fb9900fac546
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-02-04 15:57:59 +0000

Adjust circular approximation


---

diff --git a/src/polygon.h b/src/polygon.h
index 369efea..2d65302 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -34,7 +34,7 @@
 
 /* Implementation constants */
 
-#define POLY_CIRC_SEGS 40
+#define POLY_CIRC_SEGS 8
 #define POLY_CIRC_SEGS_F ((float)POLY_CIRC_SEGS)
 
 #if 0
